configure.ac

```diff
@@ -954,6 +954,8 @@ AS_IF([test "x$enable_post_processing" = "xyes"],
                          [
                            glib-2.0 >= $glib_version
                            jansson >= $jansson_version
+                           libssl >= $ssl_version
+                           libcrypto
```

Comment:
If you add these here, you'll need to add `$(BORINGSSL_LIBS)` and `$(BORINGSSL_CFLAGS)` as we do for `janus_LDADD` and `janus_CFLAGS` to all the postprocessor tools as well. That's needed because if people are configuring Janus to use BoringSSL, the lack of those variables will cause the wrong crypto library to be used (or one not to be found at all).

Just see how those variables are currently used in `Makefile.am`: there's no problem adding them even when BoringSSL is not used, as they default to empty strings and are only set if BoringSSL is enabled when configuring Janus.

Reply:
done, please check again